### ISSUE TITLE
Toggle Group context breaks the "asChild" prop

### ISSUE THREAD

**Issue by charliematters** [shadcn-ui/ui#1987]
As the title - using the asChild prop on a `<ToggleGroup>` means that keyboard navigation doesn't work on the toggle group items. Moving the context to the outer layer (and sucking up the re-renders) solves the issue

**Comment by shadcn** [comment]
This issue has been automatically closed because it received no activity for a while. If you think it was closed by accident, please leave a comment. Thank you.
